geomstats/_backend/pytorch/__init__.py

```diff
@@ -24,6 +24,9 @@
     flatten,
     float32,
     float64,
+    complex32,
```

Comment:
These complex types need to be imported by the other backends too, for consistency among the backends.

Could you modify the corresponding files, i.e.
- _backend/numpy/__init__.py
- _backend/autograd/__init__.py
- _backend/tensorflow/__init__.py
?

Thannks!